src/main/java/javi/sonarqube/ComplexMethod.java

```diff
@@ -0,0 +1,33 @@
+package javi.sonarqube;
+
+import java.util.logging.*;
+
+public class ComplexMethod {
+
+    public String intToEnglishValue(int i) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "intToEnglishValue" is 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00112)